### ISSUE TITLE
missing properties are not checked

### ISSUE THREAD

**Issue by weijarz** [facebook/flow#4594]
[Try Flow](https://flow.org/try/#0PTAEAEDMBsHsHcBQBjaBDAzh0BpUBvRUUAFzQHNQAKASgC5QMSAnASwDtLDjjlZ2mpCqAC8pABasMAOgBuAU2YZW-USLEBGUAH5QAclka9oBgYBMeoj2bySAV2bsh5KwF9EroA>)

```js
// @flow
class K {
  getTag (): string {
    return this.version === 1 ? 'v1' : 'v2'
  }
}
```

Here `this.version` has no type annotation.

**Comment by juanibiapina** [comment]
This might be related to the case I reported here: #4592

**Comment by juanibiapina** [comment]
Here's another example which intersects both our cases:

```javascript
/* @flow */

type Action = { type: "A" }

const action: Action = { type: "A", b: 1 };

action.asdfasdfasdf === "A" ? 1 : 0;
```

**Comment by dannyharding10** [comment]
This is fixed since version 0.68.0. I suggest closing this issue
